build/packages-template/bbb-export-annotations/build.sh

```diff
@@ -26,6 +26,11 @@ popd
 mkdir -p staging/usr/lib/systemd/system
 cp bbb-export-annotations.service staging/usr/lib/systemd/system
 
+#
+# Install fonts
+mv fonts/* /usr/local/share/fonts/
```

Comment:
```suggestion
cp fonts/* /usr/local/share/fonts/
```

Just to avoid any possible issues with local environment test builds